cartographer/metrics/family_factory.h

```diff
@@ -47,7 +47,7 @@ class Family {
 template <typename MetricType>
 class NullFamily : public Family<MetricType> {
  public:
-  MetricType* Add(const std::map<std::string, std::string>& labels) override {
+  MetricType* Add(const std::map<std::string, std::string>&) override {
```

Comment:
We should probably preserve the name of the unused parameter at least. Often cstyle comments (bless their hearts) are used for this. Alternately if possible using a pragma to ignore the warning might make sense here.

Reply:
This might also work if hideous: `__attribute__((unused))`

Reply:
```suggestion
  MetricType* Add(const std::map<std::string, std::string>& /*labels*/) override {
```
Like this?